packages/testing/src/common/setup.ts

```diff
@@ -5,24 +5,29 @@ import dotenv from 'dotenv';
 import type { ClerkSetupOptions, ClerkSetupReturn } from './types';
 
 export const fetchEnvVars = async (options?: ClerkSetupOptions): Promise<ClerkSetupReturn> => {
+  const { debug = false, dotenv: loadDotEnv = true, ...rest } = options || {};
```

Comment:
Nit: Why not destruct also `publishableKey` and `secretKey` here? 🤔

Reply:
@anagstef Looking back at the code, it was cleaner to do it this way such that there weren't variable naming conflicts.